crates/wasmparser/src/operators_validator.rs

```diff
@@ -910,7 +910,11 @@ impl OperatorValidator {
             Operator::BrTable { ref table } => {
                 self.check_operands_1(Type::I32)?;
                 let mut depth0: Option<u32> = None;
-                for relative_depth in table {
+                for element in table.targets() {
+                    let (relative_depth, _default) = element.map_err(|mut e| {
```

Comment:
```suggestion
                    let (relative_depth, _is_default) = element.map_err(|mut e| {
```

Reply:
`is_default` makes it clearer that this is a `bool`. at first I thought you'd always iterate over all targets and the default target but and had to look at the signature to clear this confusion.